### PR TITLE
[#1119] Author: remove unused class attribute

### LINKED ISSUE #1119: Author: class attribute 'NAME_FAILED_TO_CLONE_OR_CHECKOUT' is unused
Relevant code snippet:
https://github.com/reposense/RepoSense/blob/4d38d298cc0b7e0e03f9a8b0d7e88e2f46a31ad5/src/main/java/reposense/model/Author.java#L11-L13

Let's remove this if it is un-intentional?

### PR DESCRIPTION
**Pull request by t-cheepeng**
Fixes #1119 

```
Author class has an unused and deprecated class attribute 
`NAME_FAILED_TO_CLONE_OR_CHECKOUT`.

Let's remove the unused attribute in the Author class to make the 
code base cleaner.
```

### CODE REVIEW

**Comment by 0blivious** [comment]
Thank you for your contribution to RepoSense. Please follow our contributor guidelines and update your commit message accordingly.

**Comment by anubh-v** [comment]
@t-cheepeng, thanks for the contribution!
Please provide a PR message that describes the changes in this PR, following the guidelines [here](https://oss-generic.github.io/process/docs/FormatsAndConventions.html#commit-message)

You can refer to [this PR](https://github.com/reposense/RepoSense/pull/1141) for an example.
Note that the PR message is formatted within a code block.

**Comment by damithc** [comment]
@t-cheepeng any updates?

**Comment by t-cheepeng** [comment]
@damithc @anubh-v @0blivious Thanks for referring me to the guidelines! Please check if there are any more changes that need to be made to the commit.

File: src/main/java/reposense/model/Author.java [hunk 1]
```diff
@@ -11,7 +11,6 @@
  */
 public class Author {
```
Comment (0blivious) [inline src/main/java/reposense/model/Author.java]: can remove this blank line as well

**Comment by 0blivious** [comment]
For your PR description pls keep each line under 72 characters

**Pull request merged**